source/CMakeLists.txt

```diff
@@ -383,4 +284,103 @@ target_sources(EndlessSkyLib PRIVATE
 	text/alignment.hpp
 	text/layout.hpp
 	text/truncate.hpp
+	ui/BankPanel.cpp
+	ui/BankPanel.h
```

Comment:
A lot of stuff is in the ui directory; I feel that panels and realtime-stuff like AmmoDisplay and Radar are better of into separate directories.

Reply:
`/ui/hud/`?